proc/proc.go

```diff
@@ -704,8 +704,11 @@ func (dbp *Process) execPtraceFunc(fn func()) {
 func (dbp *Process) getGoInformation() (ver GoVersion, isextld bool, err error) {
 	vv, err := dbp.EvalPackageVariable("runtime.buildVersion")
 	if err != nil {
-		err = fmt.Errorf("Could not determine version number: %v\n", err)
-		return
+		vv, err = dbp.EvalPackageVariable("runtime/internal/sys.BuildVersion")
+		if err != nil {
+			err = fmt.Errorf("Could not determine version number: %v\n", err)
+			return
+		}
 	}
```

Comment:
Would it be better to evaluate `runtime/internal/sys.BuildVersion` first and fallback to `runtime.buildVersion`? Go 1.6 will be released fairly soon. A comment mentioning that the name changed in Go 1.6 might be useful as well.